en/docs/ipaas/micro-integrator/developing-integrations-with-integration-studio.md

```diff
@@ -0,0 +1,106 @@
+
+Choreo supports following types of MI Integrations
+
+- __Integration as an API__
+- __Event-Triggered Integration__
+- __Scheduled Integration__
+- __Manually-Triggered Integration__
+- __Composite Integration (Coming soon)__
+
+## Develop with Integration Studio
+
+[Integration Studio](https://wso2.com/integration/integration-studio/) is a graphical development environment for
+building and deploying integration artifacts in WSO2 Micro Integrator. It is an Eclipse-based IDE that provides a
+comprehensive set of tools for designing and testing integration flows and building integration artifacts. With WSO2
+Integration Studio, developers can create integration projects and develop integration artifacts in a visual,
+drag-and-drop manner, reducing the time and effort required to build and deploy integration flows. Additionally, the
+Studio provides features such as debugging, testing, and version control to help ensure the quality and reliability of
+integration flows.
+
+Creating an Integration project and Composite Exporter project in WSO2 Integration Studio allows developers to export
+their integration projects as a single deployment artifact, called a composite application. This composite application
+contains all the components and configurations required to deploy the integration project to WSO2 Micro
+Integrator. The Composite Exporter project helps to simplify the deployment process by allowing developers to package
+their integration projects as a single, self-contained artifact that can be easily deployed and managed. This can help
+improve the consistency and reliability of integration projects and simplify the overall deployment process.
+
+To get started with Integration Studio, see [WSO2 API Manager Documentation - Developing Integration Solutions](https://apim.docs.wso2.com/en/latest/integrate/develop/intro-integration-development/).
+
+## Integration Project Directory Structure
+
+Integration Project is a multi module maven project. It can contain multiple modules with WSO2 Synpase configurations
+and composite application module to export all the configurations as a single deployable artifact.
+Addition to the above OpenAPI definition file and libraries can be added optionally. OpenAPI definition can be added to
+the any location in the project and should be define it’s path when creating a component. Java libraries can be added to
+the directory created in the project root location with name libs.
+
+## OpenAPI support
+
+An [OpenAPI definition](https://spec.openapis.org/oas/v3.1.0#openapi-specification) can be developed to define your REST
+API. When a REST API is developed with integration studio and deployed on Choreo, it allows you to give the OpenAPI
+definition to attach to your Integration as an API component. Attaching an OpenAPI definition allows developers and
+consumers to understand the API.
+
+When a REST API is developed in WSO2 Integration Studio, developers can add an OpenAPI definition to the project. During
```

Comment:
Suggestion:
"When you develop a REST API in the [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/), you can add an OpenAPI definition to the project. When you create the component, you can provide a path to the OpenAPI Specification (OAS) file."